packages/app-storage/src/Selection/Modules.tsx

```diff
@@ -55,6 +55,11 @@ class Modules extends TxComponent<Props, State> {
             defaultValue={this.defaultValue}
             label={t('selected state query')}
             onChange={this.onChangeKey}
+            help={
+              t(meta && meta.documentation
```

Comment:
We don't need to add the `t` wrapper around here (these come directly from the node, so is not translatable), and would suggest simply -

`help={meta && meta.documentation && meta.documentation.join(' ')}`